.github/workflows/build.yml

```diff
@@ -0,0 +1,96 @@
+name: Build
+
+on:
+  push:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    name: Unit Tests
+
+    strategy:
+      matrix:
+        node-version:
+          - 12.x
+          - 14.x
+        os:
+          - ubuntu-latest
+          - windows-latest
+          - macOS-latest
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - name: git checkout
+        uses: actions/checkout@v2
+
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+    
+      - run: npm ci
```

Comment:
You will need to add a `postinstall` line to call lerna for this to work. Similar to what is in here: 

https://github.com/accordproject/concerto/blob/c2ecad3e5d52b0e8b1c7caf026d62abbf4f68b6f/package.json#L25-L26